spec.md

```diff
@@ -42,6 +42,57 @@
 
 *Copyright: 2022 Trust Over IP Foundation*
 
+**TOC**
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+
+**Table of Contents**
+
+- [Trust over IP (ToIP) Technology Architecture Specification](#trust-over-ip-toip-technology-architecture-specification)
```

Comment:
Could we remove this line, since that heading is the title of the spec and not a section inside it?

Reply:
yep. sounds good